app/src/main/java/com/example/theperiodpurse/ui/setting/SettingScreenLayout.kt

```diff
@@ -0,0 +1,179 @@
+           Switch(
+               checked = checked,
+               onCheckedChange = { checked = !checked},
+               modifier = modifier
+                   .fillMaxWidth()
+                   .wrapContentWidth(Alignment.End),
+               colors = SwitchDefaults.colors(
+                   uncheckedThumbColor = Color.DarkGray
+               )
+           )
+       }
+      TabOption(
+          stringResource(id = R.string.customize_notifications),
+          onTabClicked = onNotificationClicked
+      )
+       Text(
+           text = stringResource(R.string.account_settings_heading),
+           modifier = modifier.padding(
+               top = 50.dp,
+               start = 10.dp),
+           color = Color.Gray
+       )
+       TabOption(
+           text = stringResource(R.string.back_up_account),
+           onTabClicked = onBackUpClicked
+       )
+       Spacer(modifier = Modifier.padding(10.dp))
+       TabOption(
+           text = stringResource(id = R.string.delete_account),
+           onTabClicked = onDeleteClicked
+       )
+       Spacer(modifier = Modifier.padding(10.dp))
+       Text(text = "The Period Purse. All rights reserved.")
+   }
+}
+
+@Composable
+fun TrackingPreferencesRow(modifier: Modifier = Modifier){
+    Row(modifier = modifier
+        .fillMaxWidth()
+        .padding(30.dp),
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.Center) {
+        TrackingOptionButton(
+            modifier = modifier,
+            label = stringResource(R.string.mood),
+            icon = painterResource(id = R.drawable.sentiment_neutral_black_24dp)
+        )
+        TrackingOptionButton(
+            modifier = modifier,
+            label = stringResource(R.string.exercise),
+            icon = painterResource(id = R.drawable.self_improvement_black_24dp)
+        )
+        TrackingOptionButton(
+            modifier = modifier,
+            label = stringResource(R.string.cramps),
+            icon = painterResource(id = R.drawable.sick_black_24dp)
+        )
+        TrackingOptionButton(
+            modifier = modifier,
+            label = stringResource(R.string.sleep),
+            icon = painterResource(id = R.drawable.nightlight_black_24dp)
+        )
+    }
+}
+
+@Composable
+fun TrackingOptionButton(modifier: Modifier, label: String, icon: Painter) {
+    var checked by remember { mutableStateOf(false) }
+    val color = if (checked) Color.Green else Color.White
+    Column(
+        modifier = modifier
+            .padding(10.dp),
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally,) {
+        IconToggleButton(
+            checked = checked,
+            onCheckedChange = { checked = !checked},
+            modifier = Modifier.clip(RoundedCornerShape(20.dp))
+        ) {
+            Icon(
+                painter = icon,
+                contentDescription = "contentDescription",
```

Comment:
do we want "contentDescription" to be the content description of the button? Perhaps we can have something more informative like "click this to track/stop tracking symptom <label>" depending on whether this button is checked